_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,199 @@
+
+Let's look at what this means for plugin authors and users.
+
+### Lua plugins
+
+Neovim exposes its API natively to Lua via, e.g., `vim.api.nvim_open_win()`. It also provides methods for accessing vimscript (ex) commands and variables via, e.g., `vim.cmd("echo 'foo'")` and `vim.g.syntax_on`, respectively. This allows writing plugins with the same capabilities as those written in Vimscript while exploiting the performance of Lua(JIT) at core programming language tasks like loops. It is also possible to make use of Lua's own `luarocks` plugin ecosystem.
+
+Correspondingly, over the course of the 0.5 development cycle, there has been an explosion in the number of Lua plugins, ranging from rewrites of popular Vimscript plugins to completely new ones that would not have been possible in Vimscript -- often from contributors who were completely new to (neo)vim plugin development and were averse to learning Vimscript for that task. As a positive side effect of the long development cycle, many of these are already fully-featured and stable at the time of the 0.5 release!
+
+Here is a small and non-representative list of Lua plugins:
+
+- [Plenary](https://github.com/nvim-lua/plenary.nvim) -- A lua library of useful utilities for developing Neovim plugins (some of which will later be integrated into core).
+- [Packer](https://github.com/wbthomason/packer.nvim) -- A use-package like plugin manager with support for plugin dependencies, lazy-loading, and installing luarocks.
+- [Telescope](https://github.com/nvim-telescope/telescope.nvim) -- A highly extendable fuzzy finder over lists.
+- [Gitsigns](https://github.com/lewis6991/gitsigns.nvim) -- A Lua (teal) plugin for showing and interacting with changes on files a git repository (asynchronously).
+- [Nvim-compe](https://github.com/nvim-lua/completion-nvim) -- An auto completion framework for various sources, including Neovim's builtin LSP client.
+- [Nvim-dap](https://github.com/mfussenegger/nvim-dap) -- A debug adapter protocol implementation for step-through debugging of your code.
+- [Colorizer](https://github.com/norcalli/nvim-colorizer.lua) -- A high-performance color highlighter for Neovim without any external dependencies.
+- [Formatter](https://github.com/mhartington/formatter.nvim) -- A plugin for asynchronously executing external formatting tools on the current buffer or range.
+- [Hop.nvim](https://github.com/phaazon/hop.nvim) - Easy-motion without messing with your buffer.
+- [Neogit](https://github.com/TimUntersberger/neogit) - Magit but for Neovim.
+
+Not all of these are actually written in Lua: there are many other languages (some of which are typed) that compile to Lua, e.g.,
+- [Fennel(Lisp)](https://fennel-lang.org) via [Aniseed],(https://github.com/Olical/aniseed) or [Fennel-nvim](https://github.com/jaawerth/fennel-nvim),
+- [Teal](https://github.com/teal-language/tl) (a typed dialect of Lua),
```

Comment:
no need for parentheses

Reply:
I'll just remove it; the description is under the link.